code4me-jetbrains-plugin/src/main/java/me/code4me/plugin/completions/Code4MeCompletionContributor.java

```diff
@@ -177,8 +178,9 @@ protected void addCompletions(
 
             ScheduledFuture<?> dataRequest = completionCache.getTimeoutSupplier().get();
 
+            ArrayList<LookupElement> elements = new ArrayList<>();
```

Comment:
```suggestion
            // Need to add all elements in one go to avoid split
            ArrayList<LookupElement> elements = new ArrayList<>();
```